app/src/main/java/uk/org/openseizuredetector/MainActivity.java

```diff
@@ -114,26 +116,6 @@ public void onCreate(Bundle savedInstanceState) {
 
         // Initialise the User Interface
         setContentView(R.layout.main);
-        //getWindow().getDecorView().setBackgroundColor(okColour);
-
-        /* Force display of overflow menu - from stackoverflow
-         * "how to force use of..."
-         */
-        try {
-            Log.v(TAG, "trying menubar fiddle...");
-            ViewConfiguration config = ViewConfiguration.get(this);
-            Field menuKeyField =
-                    ViewConfiguration.class.getDeclaredField("sHasPermanentMenuKey");
-            if (menuKeyField != null) {
-                Log.v(TAG, "menuKeyField is not null - configuring....");
-                menuKeyField.setAccessible(true);
-                menuKeyField.setBoolean(config, false);
-            } else {
-                Log.v(TAG, "menuKeyField is null - doing nothing...");
-            }
-        } catch (Exception e) {
-            Log.v(TAG, "menubar fiddle exception: " + e.toString());
-        }
```

Comment:
It is better to undo here: fiddle only if below SDK 29.
If above, unnecessary: 3 dotted is overflow and hardware key is not there.